### DISCUSSION TITLE
Compare JuiceFS with LakeFS

### DISCUSSION THREAD

**Discussion by Suave** [juicedata/juicefs#948] [General]
[LakeFS](https://lakefs.io) - Atomic versioned data lake. Transform your object storage into a Git-like repository, lakeFS enables you to manage your data lake the way you manage your code. Run parallel pipelines for experimentation and CI/CD for your data.

What's different with JuiceFS?

**Comment by SandyXSD** [comment]
Simple conclusion:
- LakeFS: NOT POSIX-compatible. Users have to interact with it through certain CLIs or APIs. The usage pattern is more like object storage.
- JuiceFS: POSIX-compatible. Users can **mount** it and operate files through system calls. The usage pattern is more like local file systems.
